packages/xchain-radix/src/client.ts

```diff
@@ -157,16 +157,11 @@ export class RadixSpecificClient {
 
         // We need to do this because item.amount can be either string or number
         // depending on the type of resource
-        let xrdAmount: number
-        if (typeof item.amount === 'string') {
-          xrdAmount = parseFloat(item.amount) * 10 ** decimalDiff
-        } else {
-          xrdAmount = item.amount * 10 ** decimalDiff
-        }
+        const amount = typeof item.amount === 'string' ? parseFloat(item.amount) : item.amount
 
         const balance: Balance = {
           asset: asset,
-          amount: baseAmount(xrdAmount, XRD_DECIMAL),
+          amount: assetToBase(assetAmount(amount)),
```

Comment:
Is there an easy way to retrieve the decimals from the assets?

Reply:
this should be now fixed